bootstrap/postgres/configs.go

```diff
@@ -64,22 +65,30 @@ func (cr configRepository) Save(ctx context.Context, cfg bootstrap.Config, chsCo
 			e = repoerr.ErrConflict
 		}
 
-		cr.rollback("Failed to insert a Config", tx)
+		if err := cr.rollback("Failed to insert a Config", tx); err != nil {
+			return "", err
+		}
```

Comment:
error text should not start with capital letter. because it will be wrapped inside another other. 

Please wrap rollback error with original error. 


```suggestion
		if errRollback := cr.rollback("Failed to insert a Config", tx); errRollback != nil {
			return "", errors.Wrap(err,errRollback)
		}
```